samples/android/gcm-demo/README.md

```diff
@@ -0,0 +1,62 @@
+Google Cloud Messaging - Android Demo App
+==========================================
+
+This is a demo app used to demonstrate demonstrates various aspects of
+interacting with GCM on Android. Specifically, it covers:
+
+* Registering a client app
+* Sending downstream, upstream and notification messages
+* Subscribing to topics
+* Creating notification groups
+* Using the network manager to schedule tasks
+
+A full GCM implementation requires both a client implementation and a server
+implementation. For more information on GCM, including an overview and
+integration instructions, see
+[Cloud Messaging](https://developers.google.com/cloud-messaging/).
+
+## Getting Started
+
+This sample uses the Gradle build system. To build this project, use the
+"gradlew build" command or use "Import Project" in Android Studio. To use the
+app you will need a sender ID to be able to register and test client side APIs,
+and an API key to be able to test server side APIs. For more information, see
+[Implementing GCM Client on Android](https://developers.google.com/cloud-messaging/android/client).
+
+## Screenshots
+
+![Menu](samples/android/gcm-demo/screenshots/menu.png "The app's menu")
```

Comment:
The screenshot are still broken:
https://github.com/tjohns/gcm/tree/master/samples/android/gcm-demo